### PR TITLE
Use the theme a11y-dark for code blocks

### LINKED ISSUE #1840: "Getting Started" page code font is difficult to read
### Page(s) Affected

Getting started https://www.rust-lang.org/learn/get-started, possibly other pages with code

### What needs to be fixed?

The theming of code blocks is pretty difficult to read and has really low contrast, especially in sections with the white background. Almost seems like it is intentionally muted?

image

image

### Suggested Improvement

Just adjust the theme to something that stands out more

### PR DESCRIPTION
**Pull request by senekor**
closes #1840

### CODE REVIEW

File: static/styles/a11y-dark.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*!
```
Comment (Manishearth) [inline static/styles/a11y-dark.css:1]: issue: please add a license header

(saying it is under the MIT license, linking to the source, and linking to the full text of the MIT license, should be fine)

Comment (senekor) [reply static/styles/a11y-dark.css:1]: I guess the license is BSD-3-Clause?

https://github.com/highlightjs/highlight.js/blob/main/LICENSE

Comment (senekor) [reply static/styles/a11y-dark.css:1]: Unless other licenses apply to the themes and I missed it.

**Review by Manishearth — APPROVED** [review]
ah, https://github.com/ericwbailey/a11y-syntax-highlighting is under MIT but if you copied it from there I suppose BSD is appropriate. thanks!

**Pull request merged**